tests/test_cli.py

```diff
@@ -46,32 +48,98 @@ def assert_expected_file(self, result, output_path):
         except:
             pass
 
-    def test_schema_convert_cli(self, runner, helpers):
-        data_model_csv_path = helpers.get_data_path("example.model.csv")
+    @pytest.mark.parametrize(
```

Comment:
@linglp Had a great example I saw in a pull request where a comment was added to each of these giving it a brief overview of what the test case was for:
```
    @pytest.mark.parametrize(
        "node_label, node_display_name, expected_validation_rule",
        [
            # Test case 1: node label is provided
            (
                "PatientID",
                None,
                {"Biospecimen": "unique error", "Patient": "unique warning"},
            ),
            (
                "CheckRegexListStrict",
                None,
                ["list strict", "regex match [a-f]"],
            ),
            # Test case 2: node label is not provided and display label is not part of the schema
            (
                None,
                "invalid display label",
                [],
            ),
            # Test case 3: node label is not provided but a valid display label is provided
            (
                None,
                "Patient ID",
                {"Biospecimen": "unique error", "Patient": "unique warning"},
            ),
        ],
    )
```